msvc/sdk-build.props

```diff
@@ -97,7 +97,7 @@
       <SystemHeaderDeps>true</SystemHeaderDeps>
       <AdditionalOptions>-Werror -Wno-microsoft --system-header-prefix=winrt/ %(AdditionalOptions)</AdditionalOptions>
       <PreprocessorDefinitions>_WINOBJC_DO_NOT_USE_NSLOG=;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <InternalSystemIncludePaths>%(InternalSystemIncludePaths);$(IncludePath);$(MSBuildThisFileDirectory)..\include\;$(MSBuildThisFileDirectory)..\include\xplat;$(MSBuildThisFileDirectory)..\deps\prebuilt\include\;$(MSBuildThisFileDirectory)..\deps\prebuilt\include\icu;$([System.IO.Path]::Combine('$(MSBuildThisFileDirectory)..\include\Platform', '$(TargetOsAndVersion)'))</InternalSystemIncludePaths>
+      <InternalSystemIncludePaths>%(InternalSystemIncludePaths);$(IncludePath);$(MSBuildThisFileDirectory)..\include\;$(MSBuildThisFileDirectory)..\include\xplat;$(MSBuildThisFileDirectory)..\deps\prebuilt\include\;$(MSBuildThisFileDirectory)..\tools\deps\prebuilt\include\;$(MSBuildThisFileDirectory)..\deps\prebuilt\include\icu;$([System.IO.Path]::Combine('$(MSBuildThisFileDirectory)..\include\Platform', '$(TargetOsAndVersion)'))</InternalSystemIncludePaths>
```

Comment:
'scuse me?

Reply:
I removed the duplicate headers that were copied into deps\prebuilt\include\libdispatch. It doesn't make sense to have two additional copies of every header for libdispatch checked in. Tools uses them from the tools\deps\prebuilt\include folder.

Reply:
Ideally, we would build this as part tools package and then we don't need to check in the header, but that is for another day.

Reply:
The dispatch binaries and headers are published from tools package, so yes, we shouldn't have to change this props file at all.